src/client/components/search-component/index.jsx

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import './index.scss';
+import { Input, Button, Form, Dropdown, Header } from 'semantic-ui-react';
+import { DatesRangeInput } from 'semantic-ui-calendar-react';
+import moment from 'moment';
+
+class SearchComponent extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            destination: '',
+            date: '',
+            rooms: 1,
+            adults: 2,
+            children: 0
+        };
+    }
+
+    handleChange(event, {name, value}) {
+        this.setState({ [name]: value });
+    }
+
+    toggleRoomSelector(){
+        document.getElementById('room-selector').classList.toggle('hidden');
+    }
+
+    showRoomSelector(){
+        document.getElementById('room-selector').classList.remove('hidden');
+    }
+
+    hideRoomSelector(){
+        document.getElementById('room-selector').classList.add('hidden');
+    }
+
+    render() {
+        const selectOptions = [
```

Comment:
DRY principle violation. Please create a separate method, that generates numeric options based on the given from and to. The interface should look something like: `generateOptions(from, to);`